clients/imodels-client-management/src/operations/named-version/NamedVersionOperations.ts

```diff
@@ -7,6 +7,15 @@ import { OperationOptions } from "../OperationOptions";
 import { CreateNamedVersionParams, GetNamedVersionListParams, GetSingleNamedVersionParams, NamedVersionPropertiesForCreate, NamedVersionPropertiesForUpdate, UpdateNamedVersionParams } from "./NamedVersionOperationParams";
 
 export class NamedVersionOperations<TOptions extends OperationOptions> extends OperationsBase<TOptions> {
+  /**
+   * Gets Named Versions for a specific iModel. This method returns Named Versions in their minimal representation. The
```

Comment:
I think it should be **of**: Gets Named Version of a specific iModel

Reply:
Fixed: https://github.com/iTwin/imodels-clients/pull/72/commits/a2406984968badb809f88338fda6b9f58338a3e4